public/stylesheets/site/2.0/26-media-narrow.css

```diff
@@ -73,48 +72,12 @@ body .narrow-shown {
   margin-left: 45px;
 }
 
-/* 04 region dashboard */
-/* SKIN: Dash Line */
-/* DESCRIPTION: Horizontal dash inline */
-
-#dashboard {
-  clear: both;
-  float: none;
-  max-width: 100%;
-  margin: 1%;
-  padding: 0;
-  width: auto;
-}
-
-#dashboard .navigation, #dashboard .navigation li, #dashboard a, #dashboard .current {
-  text-align: left;
-  display: inline;
-  border: none;
-}
-
-#dashboard ul {
-  padding: 0;
-}
-
-#dashboard .landmark {
-  clear: none;
-  float: left;
-}
-
 /* 05 region main */
 
 #main {
   position: static;
 }
 
-#main, #main.dashboard {
-  float: none;
-  margin: auto;
-  width: auto;
-  padding-left: 0.5em;
-  padding-right: 0.5em;
-}
-
 #main.dashboard .index, form.filters, .works-index .index, .collections-index .index, .bookmarks-index .index, #inner form dd, #inner form dt {
```

Comment:
If #2275 is merged before this, we'll need to add `.fandoms-unassigned .index` after `.bookmarks-index .index`